samples/apps/smallbank/app/accounts.fbs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.	
```

Comment:
Can all the fbs tx types files be combined?

Reply:
Yes I believe they can be combined, it is a matter of how we want to create our flatbuffer files per app. The combination would result in one file with all the potential fields, only some being populated per RPC, and one serializer/deserializer. I think it is a bit cleaner to have them separate, but I don't have a very strong opinion :)